### PR TITLE
Add basic github actions workflow

### PR DESCRIPTION
**Pull request by gustavohenke**
Adding a basic GitHub workflow with testing, linting, and docs building.
Testing happens on Node 12, 14 and 16, whereas linting and docs building is only on Node 16.

They should all run in parallel.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        uses: actions/checkout@v2
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: npm
+      - name: Install dependencies
+        run: npm ci
+      - name: Build docs
+        run: npm run docs:build
```
Comment (fedeci) [inline .github/workflows/ci.yml:53]: Do we build docs just to check that nothing fails?

Comment (gustavohenke) [reply .github/workflows/ci.yml:53]: Pretty much, I think.

I don't recall why I added it there back in [`f9aec48` (#579)](https://github.com/express-validator/express-validator/pull/579/commits/f9aec483c4944dbc1c8218fb5766ba886b377542).

**Review by fedeci — APPROVED** [review]

**Comment by fedeci** [comment]
Maybe we have to also remove the old travis config file in this PR.

**Comment by fedeci** [comment]
It is running everything twice because of the double `push` and `pull_request` listeners🤔

**Comment by gustavohenke** [comment]
Yes 😃

**Comment by fedeci** [comment]
I'm not sure hot to avoid it :)

**Comment by gustavohenke** [comment]
It runs using the PR's latest commit and what would happen if it got merged onto the target branch.

It's a good thing - theoretically one could introduce a breaking change to `master`, and this PR be passing without nobody noticing until it's merged.

**Comment by fedeci** [comment]
Isn't it the default behaviour for both the hooks? I think that it is simply calling both of them.

**Pull request merged**